packages/formation/sass/modules/_va-pagination.scss

```diff
@@ -12,54 +11,122 @@
   display: flex;
   justify-content: center;
 
+  button {
+    -moz-osx-font-smoothing: auto;
+    -webkit-font-smoothing: auto;
+    background-color: transparent;
+    border: 0;
+    color: $color-link-default;
+    display: inline-flex;
+    font-size: 0.9em;
+    font-weight: 400;
+    justify-content: center;
+    line-height: 2;
+    padding: 0;
+    transition: all 0.3s ease-in-out;
+    transition-property: color, background-color;
+  }
+
+  // va-pagination-prev and va-pagination-next
+  &-prev,
+  &-next {
```

Comment:
I really like that you're writing the full name out - there have been several occasions where I've had a tricky time searching for something due to these sass shortcuts.